src/app/util/ember-compatibility-functions.cpp

```diff
@@ -600,8 +600,13 @@ CHIP_ERROR WriteSingleClusterData(ClusterInfo & aClusterInfo, TLV::TLVReader & a
     return apWriteHandler->AddStatus(attributePathParams, imCode);
 }
 
-} // namespace app
-} // namespace chip
+void MatterReportingListAttributeChangeCallback(chip::EndpointId endpoint, chip::ClusterId clusterId, chip::AttributeId attributeId,
+                                                ListOperation listOp)
```

Comment:
It doesn't quite make sense to have this API here if `ConcreteAttributePath` is eventually going to contain all these params.

I think this should wait till #11705 merges, at which point, you should remove this an update `MatterReportingAttributeChangeCallback` to take a `ConcreteAttributePath` instead (which will contain the list op as well).